packages/main/test/specs/ComboBox.spec.js

```diff
@@ -1014,4 +1014,13 @@ describe("Keyboard navigation", async () => {
 		assert.ok(await listItems[0].getProperty("selected"), "List Item should be selected");
 		assert.notOk(await listItems[1].getProperty("selected"), "List Item should not be selected");
 	});
+
+	it ("Tests disabled autocomplete(type-ahead)", async () => {
+		const input = await browser.$("#combo-without-type-ahead").shadow$("input");
+
+		await input.click();
+		await input.keys("b");
```

Comment:
In order to be sure that the popover is opened and everything is updated after the user input please add the following:

```
		await browser.waitUntil(() => popover.getProperty("opened"), {
			timeout: 200,
			timeoutMsg: "Popover should be displayed"
		});
```

check other tests in this file for this code